Trappist/src/Promact.Trappist.Web/wwwroot/app/core/http.service.ts

```diff
@@ -11,7 +11,13 @@ import 'rxjs/add/operator/toPromise';
 
 export class HttpService {
 
-    constructor(private http: Http) {
+    constructor(private http: Http, defaultOptions: RequestOptions) {
+
+        //Prevent request caching for internet explorer
+        defaultOptions.headers.append('Cache-control', 'no-cache');
```

Comment:
styel.css and bootstrap were being taken from the cache for http request in internet explorer
so to prevent this I had to no-cache it